test/eventbus.localstack.test.ts

```diff
@@ -63,6 +63,8 @@ localstackTestSuite("eventBusStack", (testResource) => {
           });
         }
       );
+      bus.resource.grantPutEventsTo(role);
+      table.resource.grantReadData(role);
```

Comment:
Oh do we now need to grant permissions to the role used by the tests?

Reply:
Not necessary for this iteration - but perhaps we could use integrations here? Output integration which produces a client on the other side.

Reply:
We totally could generate the test script using the serializer and integrations. Would be an interesting experiment. 

But yeah, the new annoyance is that our tests will need to make real AWS calls using a real role that actually needs permissions to make the calls.

The other option is that we make catch all roles for the tests, but I like the idea of being least privilege to set a good example. 

Luckily, the core resource like sfn and lambda can be wired using the core test logic, any extra resources will need to grant explicitly (until we start to use integrations).

Reply:
https://github.com/functionless/functionless/issues/485